src/app/clusters/bindings/BindingManager.cpp

```diff
@@ -0,0 +1,207 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/clusters/bindings/BindingManager.h>
+#include <app/util/binding-table.h>
+
+namespace chip {
+
+BindingManager BindingManager::sBindingManager;
+
+CHIP_ERROR BindingManager::CreateUnicastBinding(FabricIndex fabric, NodeId node)
```

Comment:
This naming is still not making sense to me.  The binding is already created: it's in the binding table, etc.  All we're doing now is setting up a connection to it, just in case...  That's why I suggested calling this `UnicastBindingCreated`: it's a notification that the binding has been created and some action should perhaps be taken.